docs/roles/system_manager/manage_users.md

```diff
@@ -64,6 +64,40 @@ Once you're certain that you're adding a new user, make sure that the following
 - Be particularly careful never to use them to log in to any user-accessible VMs (such as the SRDs)
 ```
 
+### {{woman}} {{man}} Adding users to security group(s)
+
+Users that didn't have a `GroupName` specified in the `user_details_template.csv`, will need to be added manually, before they can access an SRE.
```

Comment:
This is a useful tip, but I think the instructions here are more general than fixing this one problem.

Maybe something like this would be better as a warning further up the page where editing the csv is discussed? At the moment, it says this is optional, which is true, but it sounds like users are unable to work until they are added to an appropriate group.